docs/sources/upgrade-guide/upgrade-v11.2/index.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Guide for upgrading to Grafana v11.2
+keywords:
+  - grafana
+  - configuration
+  - documentation
+  - upgrade
+  - '11.2'
+title: Upgrade to Grafana v11.2
+menuTitle: Upgrade to v11.2
+weight: 1000
+---
+
+# Upgrade to Grafana v11.2
+
+{{< docs/shared lookup="upgrade/intro.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+
+{{< docs/shared lookup="back-up/back-up-grafana.md" source="grafana" version="<GRAFANA_VERSION>" leveloffset="+1" >}}
+
+{{< docs/shared lookup="upgrade/upgrade-common-tasks.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+
+## Technical notes
```

Comment:
Is there content expected or should we remove this or is it OK to leave an empty heading?

Reply:
we typically leave it tbh